packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -154,15 +154,15 @@ export const buildConnectorsQueryByQualificationLabel = label => ({
   }
 })
 
-export const buildAccountsQueryBySlug = (slug, enabled = true) => ({
+export const buildAccountsQueryBySlugs = (slugs, enabled = true) => ({
   definition: () =>
     Q(ACCOUNTS_DOCTYPE)
       .where({
-        account_type: slug
+        account_type: { $in: slugs }
```

Comment:
nit: $in / $or operators are pretty bad since they will be evaluated in memory. I don't know if you can do better, but let's call @paultranvan to the rescue. (this is a nit since `io.cozy.accounts` is not a big db)

Reply:
Oui en effet, j'avais bien ça en tête, mais vu la taille de la DB `io.cozy.accounts`, je ne suis pas sûr que tout récupérer et faire le filtre en front soit plus intéressant 🤷‍♂️ 
Ou à voir si il y a une autre approche rapide...
Attendons notre spécialiste ! 🦸

Reply:
Mon avis de spécialiste est le même que le votre :wink: 
Une alternative serait d'envoyer n (n=slugs.length) requêtes en parallèle pour évaluer un slug à chaque fois, mais je ne pense pas que ça vaille le coup vu le volume relativement maîtrisé de la bd